.changeset/three-steaks-exist.md

```diff
@@ -0,0 +1,19 @@
+---
+"@astrojs/react": minor
+---
+
+Adds experimental support for disabling streaming
```

Comment:
```suggestion
Adds experimental support for disabling streaming

This is useful to support libraries that are not compatible with streaming such as Stitches.js. To disable streaming for all React components in your project, set `experimentalDisableStreaming: true` as a configuration option for `@astrojs/react`:
```
Not a requirement, but for a minor release, it's fine to add a little more about the feature here! While this doesn't make a great *code* comment, you might want to mention a benefit here in the release notes!

Also might be worth a note/reminder that this turns off ALL streaming for ALL components (I'm assuming? This doesn't make it appear configurable per-component) in case that has effects in someone's project.

Reply:
I'd say "some CSS-in-JS libraries" or something like that, rather than mentioning Stitches.js specifically, as it's likely to be a general issue there.